examples/mdns_server.rs

```diff
@@ -34,7 +32,7 @@ async fn main() -> Result<(), Error> {
             Arg::with_name("server")
                 .required_unless("FULLHELP")
                 .takes_value(true)
-                .default_value(DEFAULT_DEST_ADDR)
+                .default_value("0.0.0.0:5353")
```

Comment:
so, we can't use "224.0.0.251:5353" for multicast listening? Is that limitation in Windows or Mac?

Reply:
We use the INADDR_ANY `0.0.0.0` so as to bind to all interfaces both on windows and unix. Specifying `224.0.0.251:5353` wont bind as it is not a valid address to any interface.